### ISSUE TITLE
Hello sir, i have an issue using the ""prior reservation method with 200 photos""

### ISSUE THREAD

**Issue by LIQUIDMIND111** [TheLastBen/fast-stable-diffusion#67]
Since I only have the subjects photos, when collab tried to generate the "CLASS photos of a WOMAN", i get NSFW, and it mentions, _A BLACK SCREEN WILL BE GENERATED_ instead, so I cannot train well my models with this issue.

SO I ask, is there a way to turn OFF NSFW filter just for training with PRIOR RESERVATION METHOD? Some of my models had a bikini, but it takes a LOT to generate a CLASS photo, so then it says CANNOT BE USED, lol, please what do you recommend doing? Can you add a code for NSFW filter removal? Or what?

**Comment by TheLastBen** [comment]
Hello, I will turn it off shortly.
Tip: you can upload the class images to your gdrive before running the colab, then use the path to that folder in the settings, it will save you time and enhance the quality

**Comment by LIQUIDMIND111** [comment]
i see. ok

**Comment by LIQUIDMIND111** [comment]
> Hello, I will turn it off shortly. Tip: you can upload the class images to your gdrive before running the colab, then use the path to that folder in the settings, it will save you time and enhance the quality

good day sir:

Hello sir, 2 questions, if I want to upload myself the CLASS photos OF MEN, what photos do i need to look for on Google? full body men? Face shots? Different angles? Do i also have to crop them to 512x512 1024x1024 ???

LAST question is, people are mentioning on YouTube comments that " for every picture you upload for training, it needs 100 steps", so if you are training a model using 15 photos, you need to use 1500 steps??? And if i myself I'm using 50 photos of me, i need to train 100x50 = 5000 steps???? Is this accurate for you?

**Comment by TheLastBen** [comment]
- If you want a photo of a single male person, you should upload square photos of a single male person (photo dimensions : H=W eg: 512x512 or 1024x1024 or 2048x2048....)

- Yes, the more steps the better, but it's better to try many options, I trained it on one single photo of a cat, 30 steps, and I got good results

**Comment by LIQUIDMIND111** [comment]
> * If you want a photo of a single male person, you should upload square photos of a single male person (photo dimensions : H=W eg: 512x512 or 1024x1024 or 2048x2048....)
> * Yes, the more steps the better, but it's better to try many options, I trained it on one single photo of a cat, 30 steps, and I got good results

yes, i read someone saying that with 100 steps it got great s\results... hummm interesting.,. Thanks,

**Comment by LIQUIDMIND111** [comment]
> * If you want a photo of a single male person, you should upload square photos of a single male person (photo dimensions : H=W eg: 512x512 or 1024x1024 or 2048x2048....)
> * Yes, the more steps the better, but it's better to try many options, I trained it on one single photo of a cat, 30 steps, and I got good results

i don't know what's happening, but ckpt files look a bit different after you change the precision to half.... 

Quality does not look as good as the first models that were 4GB in size//// im my case i would prefer the 4gb full precision, anyway, AITEPRENEUR from YouTube, also mentions that DREAMBOOTH MODELS made in RUNPOD are better quality than the ones being made on any collab notebook.... do you know why can that be? Does the conversion to CKPT change the quality of model? or ONLY the training process being used in run pod might be different??? any clues?

**Comment by TheLastBen** [comment]
The training is made on half precision, so the CKPT would behave the same if it were a full precision model.

As long as the code is the same, the quality would be the same, no matter the platform used

for the prompt you should always include the instance name in it

**Comment by LIQUIDMIND111** [comment]
> The training is made on half precision, so the CKPT would behave the same if it were a full precision model.
> 
> As long as the code is the same, the quality would be the same, no matter the platform/hardware used
> 
> for the prompt you should always include the instance name in it

ok thanks, and what about the learning rate? i have read online MANY things, like the best rate is 2e-6 other people mention its 2e-4 etc.... i never change yours, since one time i got a messed model, but what would happen if i use 2e-6 instead? the learning rate would be slower and i need MORE steps also? or what happens?
